### PR TITLE
chore(v3): debug

### PR DESCRIPTION
**Pull request by jayzhang**

### CODE REVIEW

**Review by nliu-ms — APPROVED** [review]
approve for app manifest component.

File: packages/fx-core/src/common/local/projectSettingsHelper.ts [hunk 1]
```diff
@@ -14,20 +14,34 @@ import {
...
   public static isSpfx = (projectSettings: ProjectSettings | undefined): boolean =>
-    (projectSettings?.solutionSettings as AzureSolutionSettings)?.hostType ===
-    HostTypeOptionSPFx.id;
+    isV3()
+      ? isSPFx(projectSettings as ProjectSettingsV3)
+      : (projectSettings?.solutionSettings as AzureSolutionSettings)?.hostType ===
+        HostTypeOptionSPFx.id;
 
   public static includeFrontend(projectSettings: ProjectSettings | undefined): boolean {
+    if (isV3()) return hasTab(projectSettings as ProjectSettingsV3);
```
Comment (kuojianlu) [inline packages/fx-core/src/common/local/projectSettingsHelper.ts:37]: If the project is SPFx, what will the result of `hasTab` be?

Comment (jayzhang) [reply packages/fx-core/src/common/local/projectSettingsHelper.ts:37]: hasTab is true for SPFx Tab

Comment (kuojianlu) [reply packages/fx-core/src/common/local/projectSettingsHelper.ts:37]: Currently, `includeFrontend` means including Azure Tab, not SPFx Tab.

**Review by hund030 — APPROVED** [review]

File: packages/fx-core/src/common/local/projectSettingsHelper.ts (hunk 1 above)
Comment (jayzhang) [reply packages/fx-core/src/common/local/projectSettingsHelper.ts:37]: OK, I will replace it with hasAzureTab()

**Review by kuojianlu — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

**Review by hund030 — DISMISSED** [review]

**Review by xzf0587 — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**